### DISCUSSION TITLE
how to filter ranged date?

### DISCUSSION THREAD

**Discussion by HadziqM** [pocketbase/pocketbase#1270] [Q&A] [answered]
it seem filter grouping only work one for each variable,  but fail if i group two of the same variable
for example:
http://127.0.0.1:8090/api/collections/user/records?filter=(created<'2022-12-16'&&created>'2022-12-10')
this will return {code:400,message:"Invalid filter parameters.",data:{}}
is there a way to do this?, my solution is only listing all items then filter it myself

**Comment by ganigeorgiev** [comment, marked as answer]
I cannot reproduce the issue. The above filter is working fine for me.
My guess is that you may need to url encode it, aka.:

```
http://127.0.0.1:8090/api/collections/user/records?filter=%28created%3C%272022-12-16%27%26%26created%3E%272022-12-10%27%29
```
(_the SDKs will encode the query parameters for you_).

**Reply by HadziqM** [reply]
Yeah seem like the parenthesis is not parsed on web by default, and thanks thats work, btw i m working on rust so there is no sdk.
